### PR TITLE
Update ut_seattlelib_time_interface_updatetime_before_registermethod.r2p...

### PR DESCRIPTION
**Pull request by karthikgvss**
...y
create a else block to fail the test if it does not raise exception for not registering time method

### CODE REVIEW

File: tests/ut_seattlelib_time_interface_updatetime_before_registermethod.r2py [hunk 1]
```diff
@@ -12,3 +12,6 @@ try:
 except time_interface.TimeError:
   # That's what we expect
   pass
+else:
+  # we have to fail this if we try didnt raise a exception for not registering time method
+  log("[FAIL]: Did not raise exception for not registering time method (ntp/tcp) \n")
```
Comment (vladimir-v-diaz) [inline tests/ut_seattlelib_time_interface_updatetime_before_registermethod.r2py]: I think it's clearer if you include in the message the exception that should have been raised.
Suggestion: "time_interface.time_updatetime() succeeded but should have raised time_interface.TimeError"

Comment (vladimir-v-diaz) [inline tests/ut_seattlelib_time_interface_updatetime_before_registermethod.r2py]: Note: Mentioning the time_register_method() also helps.

**Comment by karthikgvss** [comment]
Thanks Vlad. Yes, That way its more clear. Made the changes to the comment. Please check once.

File: tests/ut_seattlelib_time_interface_updatetime_before_registermethod.r2py [hunk 2]
```diff
@@ -12,3 +12,6 @@ try:
 except time_interface.TimeError:
   # That's what we expect
   pass
+else:
+  # we have to fail this if we try didnt raise a exception for not registering time method
+  log("[FAIL]: The call to time_interface.time_updatetime() method has succeeded but instead it should have raised time_interface.TimeError exception.  This exception has to be raised if time_register_method() which registers the (tcp / ntp) method is not called prior to calling time_interface.time_updatetime() method \n")
```
Comment (vladimir-v-diaz) [inline tests/ut_seattlelib_time_interface_updatetime_before_registermethod.r2py]: Apart from this string that wraps over multiple lines, this looks mergeable (once you fix the long string :).

Test results:
$ python utf.py -f ut_seattlelib_time_interface_updatetime_before_registermethod.r2py
Testing module: seattlelib
	Running: ut_seattlelib_time_interface_updatetime_before_registermethod.r2py [ PASS ]
$ vi ut_seattlelib_time_interface_updatetime_before_registermethod.r2py

Comment (karthikgvss) [inline tests/ut_seattlelib_time_interface_updatetime_before_registermethod.r2py]: Made the changes and merged it

On Wed, Jan 28, 2015 at 2:30 PM, Vladimir Diaz <notifications@github.com>
wrote:

> In
> tests/ut_seattlelib_time_interface_updatetime_before_registermethod.r2py
> <https://github.com/SeattleTestbed/seattlelib_v2/pull/160#discussion_r23715258>
> :
>
> > @@ -12,3 +12,6 @@ try:
> >  except time_interface.TimeError:
> >    # That's what we expect
> >    pass
> > +else:
> > +  # we have to fail this if we try didnt raise a exception for not registering time method
> > +  log("[FAIL]: The call to time_interface.time_updatetime() method has succeeded but instead it should have raised time_interface.TimeError exception.  This exception has to be raised if time_register_method() which registers the (tcp / ntp) method is not called prior to calling time_interface.time_updatetime() method \n")
>
> Apart from this string that wraps over multiple lines, this looks
> mergeable (once you fix the long string :).
>
> Test results:
> $ python utf.py -f
> ut_seattlelib_time_interface_updatetime_before_registermethod.r2py
> Testing module: seattlelib
> Running:
> ut_seattlelib_time_interface_updatetime_before_registermethod.r2py [ PASS ]
> $ vi ut_seattlelib_time_interface_updatetime_before_registermethod.r2py
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SeattleTestbed/seattlelib_v2/pull/160/files#r23715258>
> .
>



-- 


Thanks and regards,
Karthikeya Garimella
Graduate Student - Computer Engineering
Electrical & Computer Engineering Dept.,
New York University - Polytechnic School of Engineering
Mobile - (917) 821-7478

**Pull request merged**